app/views/posts/index.html.erb

```diff
@@ -1 +1,26 @@
-<h1>This is lits posts<h1>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+  <head>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;700&display=swap" rel="stylesheet">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+  </head>
+
+  <body>
+    <div class="root-container posts-comments-container">
+      <%= render 'shared/user_info', user: @user %>
+      <div class="posts-comments">
+        <% @posts.each_with_index do |post, post_no| %>
+          <div class="post-comments">
+            <%= render 'shared/post_info', post: post, post_no: post_no %>
+            <div class="comments">
+              <% post.comments.each do |comment| %>
+                <%= render 'shared/comment', comment: comment %>
+              <% end %>
+            </div>
```

Comment:
- Can you render the 5 most recent comments? Project requirements
![3](https://github.com/Paul-tes/Blog-App/assets/63972670/ef6dcfb0-5c9a-44d0-9f26-f4c7351820ce)